runtimes/nodejs/actions/initialHtml.js

```diff
@@ -33,12 +62,11 @@ function getHtml(theSignedUrlPut, theSignedUrlGet) {
         <h4> Before uploading a file:</h4>
         <ul>
             <li>Create Cloud Object Storage HMAC Credentials ({hmac:true} on credential creation)</li>
-            <li>Create a bucket on Cloud Object Storage</li>
-            <li>Add the bucket name as a parameter to the get-signed-url action in the cloud-object-storage package</li>
-            <li>Add CORS policy to bucket</li>
+            <li>Create a bucket in Cloud Object Storage Service</li>
```

Comment:
should this be "Create a bucket in the Cloud Object Storage Service"?

Reply:
yep, thanks.